src/engraving/rw/read410/tread.cpp

```diff
@@ -2698,6 +2698,8 @@ bool TRead::readProperties(Chord* ch, XmlReader& e, ReadContext& ctx)
         TRead::read(ss, e, ctx);
         ch->add(ss);
     } else if (TRead::readProperty(ch, tag, e, ctx, Pid::STEM_DIRECTION)) {
+    } else if (tag == "hookReverse") {
+        ch->setHookReversed(e.readBool());
```

Comment:
Maybe this needs to get added to read302.cpp too, in order to be able to read this from 3.7 scores